### ISSUE TITLE
Add option to copy URL to clipboard instead of opening it on double-click

### ISSUE THREAD

**Issue by TheWolfH** [keepassxreboot/keepassxc#4717]
## Summary
Keepass provides a configuration option to copy entry URLs to the clipboard instead of opening them in the default browser when double-clicking the entry URL. I think this would be a good addition to KeePassXC as well.

## Context
In my case, I prefer to open many links in either specific browser versions (i.e., beta) or specific windows (i.e., incognito mode) instead of the system default browser. I assume there are many more use cases, though.

**Comment by edneville** [comment]
@TheWolfH, I added ctrl-alt-u as a [PR](https://github.com/keepassx/keepassx/pull/230) for keepassx. Then I came here to do the same. However, upon looking in the dev branch this already exists as ctrl-u, introduced in:
```
commit 3a2f38789273610e9896359eac1fa46b08f77f9e
Author: Felix Geyer <debfx@fobos.de>
Date:   Fri Jul 27 18:38:52 2012 +0200

    Add action to open the entry URL.

    Closes #37
```

Or did you want the URL to copy through double click? If so, I think that would change behaviour that others may have become accustomed to.

**Comment by edneville** [comment]
Based on title change, I'd say this is already implemented with ctrl-u.

**Comment by droidmonkey** [comment]
No it is not...

**Comment by TheWolfH** [comment]
@edneville Yeah, I know I can copy the URL to the clipboard via ctrl+u. However, the desired behavior would indeed be copying the URL to the clipboard when double-clicking the URL column. As noted in the initial post, Keepass provides a config option to enable this behavior instead of the default behavior of opening the URL in the system default browser.

**Comment by iqlogic** [comment]
Any news about that? Double-click and copy URL without opening URL would be much appreciated 😊 Many thanks

**Comment by xeverse** [comment]
+1

**Comment by 2GiSBR** [comment]
Hello,
Like the author of the post mentionned, it is known that you can copy the URL with the CTRL + U shortcut, but the idea would be to be able to choose (like for the password and username) to copy the URL in the clipboard instead of opening it.
It is especially useful when you have several webbrowser profiles opened, or when you work with multiple Virtual machines.
Regards.

**Comment by mischlrebl** [comment]
shift - double-click on the URL copies it already to the clipboard

**Comment by juzu-o** [comment]
I didn't notice my #12321 issue was duplicate of this.
Since this feature request seems to have been open for a while, would this Copilot AI created solution be of interest?
Commit here:
https://github.com/juzu-o/keepassxc/commit/5c45e9616dd5e58f2ed7bba4faae636490370560

This adds drop-down/combo box menu option 'Double-click action for URL' to Settings, which allows user to select preferred action when double-clicking on URL:
<img width="978" height="721" alt="Image" src="https://github.com/user-attachments/assets/af612bf9-8384-4cf3-8eb0-c4934dab1c79" />

This menu option replaces the current 'Open browser on double clicking URL field in entry view' option.

**Comment by droidmonkey** [comment]
Make a pr for it please
